components/Buttons/examples/FloatingButtonExampleSwiftViewController.swift

```diff
@@ -0,0 +1,175 @@
+//
+//  FloatingButtonExampleSwiftViewController.swift
+//  MaterialComponentsExamples
+//
+//  Created by Rob Moore on 12/1/17.
+//
+
+import UIKit
+
+class FloatingButtonExampleSwiftViewController: UIViewController {
+
+  let floatingButtonPlusDimension = CGFloat(24)
+  let miniFloatingButton : MDCFloatingButton
+  let defaultFloatingButton : MDCFloatingButton
+  let largeIconFloatingButton : MDCFloatingButton
+
+  override init(nibName nibNameOrNil: String?, bundle nibBundleOrNil: Bundle?) {
+    miniFloatingButton = MDCFloatingButton(frame: .zero, shape: .mini)
+    defaultFloatingButton = MDCFloatingButton()
+    largeIconFloatingButton = MDCFloatingButton()
+    super.init(nibName: nibNameOrNil, bundle: nibBundleOrNil)
+  }
+
+  required init?(coder aDecoder: NSCoder) {
+    miniFloatingButton = MDCFloatingButton(frame: .zero, shape: .mini)
```

Comment:
lgtm! just small nit: due to initialization being the same for both inits, i would probably go down the route of initializing in the declaration. Usually if it is immutable and doesn't rely on `self` my preference is initializing in declaration (but that's just a preference).

Reply:
Done